runner_test.go

```diff
@@ -32,8 +33,23 @@ func TestRunner_appendSecrets(t *testing.T) {
 					"zed": secrets[1],
 				},
 			},
-			keyNames: []string{"kv_foo_bar", "kv_foo_zed"},
+			keyNames: []string{"prefix_kv_foo_bar_sufix", "prefix_kv_foo_zed_sufix"},
```

Comment:
I'd prefer it if we kept the 'kv1 secret' test as is and add another new one with the prefix/suffix test.

Also this (new) test and the one below should have different names and not reuse the 'kv1 secret' one.

Thanks.